kinto_megaphone/heartbeat.py

```diff
@@ -0,0 +1,25 @@
+from pyramid.config import ConfigurationError
+from pyramid.settings import aslist
+
+
+class MegaphoneHeartbeat(object):
+    def __init__(self, client):
+        self.client = client
+
+    def __call__(self, _request):
+        data = self.client.heartbeat()
+        return data['database'] == 'ok' and data['status'] == 'ok'
```

Comment:
Maybe we shouldn't introspect so much the heartbeat content. Officially, if response.status == 200 then it's OK